src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -3039,6 +3039,14 @@ public override BoundNode VisitFunctionPointerInvocation(BoundFunctionPointerInv
             return null;
         }
 
+        public override BoundNode VisitUnconvertedAddressOfOperator(BoundUnconvertedAddressOfOperator node)
+        {
+            // This is not encountered in correct programs, but can be seen if the function pointer was
```

Comment:
>but can be seen if the function pointer was [](start = 60, length = 43)

This disagrees with a comment in BoundNodes.xml: "should not survive initial binding." The node either survives the initial binding and the comment in BoundNodes.xml should be adjusted, or it doesn't and this method should throw.